Feature/mc 12978 create workload (#279)
cloudops/cloudmc-api-docs: source/includes/stackpath/_workloads.md

`isRemoteManagementEnabled`<br/>*boolean* | Specify if remote management is enabled on workload instance or not.
`image`<br/>*string* | The workload's instance operating system image.

<!-------------------- CREATE A WORKLOAD -------------------->
### Create a workload
```shell
curl -X POST \
    -H "MC-Api-Key: your_api_key" \
    -d "request_body" \
    "https://cloudmc_endpoint/v1/services/stackpath/test-area/workloads"
```

> > Request body example for a VM:

Reviewer: Just a small formatting change to match [our docs template](https://raw.githubusercontent.com/cloudops/cloudmc-api-docs/cmc-dev/.github/PULL_REQUEST_TEMPLATE.md) 📜 

To be consistent with the rest of the docs, we could put both request examples in the same code block (with format js) and the response example in another.

> Request body examples:
```js
// create a vm
{
   name: my-vm,
   slug: my-vm-slug,
   ...
}

// create a container
{
   name: my-container,
   slug: my-container-slug,
   ...
}
```

> The above commands return a JSON structured like this:
```json
{
  "taskId": "ef70cafa-0544-4709-a66a-c68595ee105a",
  "taskStatus": "SUCCESS"
}

```

Author: fixed in next commit

   "persistenceStorageSize":1,
   "deploymentName":"wi-root-ion",
   "deploymentInstancePerPops":1,
   "enableAutoScaling":false,
   "deploymentPops":"YYZ"
}
```
> Request body example for a Container:
```json
{
   "name":"w-user-pah",
   "type":"CONTAINER",
   "image":"nginx:latest",
   "addImagePullCredentialsOption":true,
   "containerUsername":"test.username",
   "containerPassword":"test-password",
   "containerServer":null,
   "containerEmail":null,
   "environmentVariableKey": null,
   "environmentVariableValue": null,
   "secretEnvironmentVariableKey":null,
   "secretEnvironmentVariableValue":null,
   "vpc":"Default",
   "addAnyCastIpAddress":false,
   "publicPorts":"80-200",
   "protocol":"UDP",
   "specs":"SP-3",
   "persistenceStoragePath":null,
   "persistenceStorageSize":1,
   "deploymentName":"wi-root-pxa",
   "deploymentInstancePerPops":1,
   "deploymentPops":"FRA",
   "enableAutoScaling":true,
   "cpuUtilization":50,
   "minInstancesPerPop":1,
   "maxInstancesPerPop":2
}
```
<code>POST /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/workloads</code>

> Create a new workload

Reviewer: Should be `Create a new workload in a given [environment](#administration-environments).`

Author: good catch. Thanks Pat

 `image`<br/>*string* | Either the location of a Docker image to run as a container or the image to use for the virtual machine. If for a virtual machine, this is in the format of /[:]. If the image tag portion is omitted, 'default' is assumed which is the most recently created, ready, and non-deprecated image of that slug. A set of common images is present on the 'stackpath-edge' stack.
 `vpc`<br/>*string* | The virtual private cloud option for now supports default vpc only.
 `protocol`<br/>*string* | Supported protocols are: `TCP`, `UDP` and `TCP_UDP`.
 `firstBootSshKey(s)`<br/>*string* | If creating a VM-based workload, SSH keys are required. Multiple SSH keys can be separated by commas.
 `specs`<br/>*string* | Specification type for resources which are allocated to each instance in a workload. Supported specifications are `SP-1 (1 vCPU, 2 GB RAM)`,`SP-2 (2 vCPU, 4 GB RAM)`,`SP-3 (2 vCPU, 8GB RAM)`,`SP-4 (4 vCPU, 16 GB RAM)`,`SP-5 (8 vCPU, 32 GB RAM)`.
 `deploymentName`<br/>*string* | The name of the deployment.
 `deploymentPops`<br/>*string* | The point of presence of a deployment. In the format [A-Z][A-Z][A-Z].

>  `enableAutoScaling` <br/>*boolean* | Specify if you would like to enable autoscaling. If enabled, the following fields are required.

Reviewer 2: ```suggestion
 `enableAutoScaling` <br/>*boolean* | Specify if you would like to enable autoscaling. If enabled, then `cpuUtilization` , `minInstancesPerPop` and `maxInstancesPerPop` are required.
```

We might want to add more fields after `maxInstancesPerPop` and this way it won't confuse API consumers if those newly added fields after `enableAutoScaling` are required by `enableAutoScaling` or not.

 `cpuUtilization` <br/>*int* | Specify the percentage of CPU utilization.
 `minInstancesPerPop` <br/>*int* | The minimum number of instances per PoP.
 `maxInstancesPerPop` <br/>*int* | The maximum number of instances per PoP.
 
 Optional | &nbsp;
 ------- | -----------
 `addAnyCastIpAddress`<br/>*boolean* | Option to AnyCast IP Address.
 `slug`<br/>*string* | A workload's programmatic name. Workload slugs are used to build its instances names. If not provided, defaults to workload's name.
 `publicPorts`<br/>*string* | A single port, such as 80 or a port range, such as 1024-65535.
 `persistenceStoragePath`<br/>*string* | The path in an instance to mount a volume.
 `persistenceStorageSize`<br/>*int* | The size of the mounted volume (in GB).

>  `addImagePullCredentialsOption` <br/>*boolean* | When creating a Container based workload, use this option to pull a container image.

Reviewer 2: ```suggestion
 `addImagePullCredentialsOption` <br/>*boolean* | It is used to indicate if additional credentials to pull container image are provided or not.
```

Author: made the change in next commit 👍
